<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DES</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="input_Box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>250</width>
      <height>250</height>
     </rect>
    </property>
    <property name="title">
     <string>输入框</string>
    </property>
    <widget class="QPlainTextEdit" name="input_text">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="output_Box">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>0</y>
      <width>250</width>
      <height>250</height>
     </rect>
    </property>
    <property name="title">
     <string>输出框</string>
    </property>
    <widget class="QTextEdit" name="output_text">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="key_Box">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>密钥</string>
    </property>
    <widget class="QPlainTextEdit" name="key_text">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="code_Box">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>选择模式</string>
    </property>
    <widget class="QRadioButton" name="encode_RB">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>加密</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="decode_RB">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>解密</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="run_button">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>160</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
